docs/site/package.json

```diff
@@ -9,7 +9,9 @@
   },
   "scripts": {
     "start": "node webpack.dev.server.js",
-    "build": "cross-env NODE_ENV=production webpack --config webpack.prod.config.js"
+    "build": "npm run clean&& cross-env NODE_ENV=production webpack --config webpack.prod.config.js",
+    "clean": "rimraf build",
+    "lint" :"eslint src --fix --color"
```

Comment:
There is a linting issue : `" :"`.
What the use case for this command? We already have a `lint` command at the root.